src/phrases/phrases-en.ts

```diff
@@ -540,6 +540,10 @@ const phrases = {
       illegalSlug:
         'Use of illegal characters in permalink! Legal characters are numbers and letters, plus dashes.',
     },
+    isFrontpageArticle: {
+      description:
+        'Here you control if the article is to be portrayed as a frontpage article for you in the editor',
```

Comment:
Noe usikker på ordlegging her. Kunne sikkert sluppet unna med `Controls whether the article is displayed as a frontpage article in the editor.`